### DISCUSSION TITLE
Can't log in to app on deployed site. Error code: 400. "API key not valid"

### DISCUSSION THREAD

**Discussion by nashvegasnate** [nss-evening-cohort-14/evening-client-side#224] [Help Me!] [answered]
Nathan Keith 

Issue tickets searched:
#177 
#193 
#125

Issue description:
I have deployed my app to Netlify, however, when I try to sign in, it won't authenticate and I get this error:
![Screen Shot 2021-05-10 at 2 40 17 PM](https://user-images.githubusercontent.com/76710923/117715492-f5fe5600-b19d-11eb-8e70-052bfb7b569c.png)

What I have Googled:
- https://stackoverflow.com/questions/49234707/reactjs-and-node-error-400-bad-request
- https://kinsta.com/knowledgebase/400-bad-request/
- https://github.com/firebase/firebaseui-web-react/issues/57

What I have tried:
- re-entering keys in FIrebase and .env files
- re-entering environmentals in Netlify
I have followed the instructions extremely closely in https://github.com/nss-nightclass-projects/REACT-Deployment-Netlify but am still getting the same error.

Netlify link: https://react-highlander-roster.netlify.app/
Link to branch: https://github.com/nashvegasnate/react-sports-roster/tree/main

**Comment by TrinityChristiana** [comment, marked as answer]
Looking at the deployed version, the text "REACT_APP_API_KEY" is in the API key. Do you see that locally?
![Screen Shot 2021-05-10 at 7 22 18 PM](https://user-images.githubusercontent.com/31781724/117740085-4c32bf80-b1c5-11eb-8c86-e1b016e1b968.png)

**Comment by nashvegasnate** [comment]
The issue was in fact the above-mentioned cut/paste error. It had been taken care of earlier in the day, however, it required a re-deployment in order for the app to work properly.
Thanks, Trinity!
